src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -82,6 +86,8 @@ function getNowPlayingBarHtml() {
 
     html += `<button is="paper-icon-button-light" class="btnAirPlay mediaButton" title="${globalize.translate('AirPlay')}"><span class="material-icons airplay" aria-hidden="true"></span></button>`;
 
+    html += `<button is="paper-icon-button-light" class="openLyricsButton mediaButton" title="${globalize.translate('Lyrics')}"><span class="material-icons mic_external_on" aria-hidden="true"></span></button>`;
```

Comment:
Why not use the lyrics icon?

Reply:
`lyrics` icon conflicts with new `lyrics` class. But it looks better.